pkg/mdformatter/mdformatter_test.go

```diff
@@ -52,6 +53,7 @@ func TestCheck_NoTransformers(t *testing.T) {
 
 	exp, err := ioutil.ReadFile("testdata/not_formatted.md.diff")
 	testutil.Ok(t, err)
+	fmt.Println(diff.String())
```

Comment:
Can this be removed? Diff gets printed here irrespective of test passing/failing.  🙂

Reply:
ah forgot to change